packages/account/src/utils/Types.ts

```diff
@@ -142,6 +142,7 @@ export type BuildUserOpOptions = {
   nonceOptions?: NonceOptions;
   forceEncodeForBatch?: boolean;
   paymasterServiceData?: PaymasterUserOperationDto;
+  simulationType?: SimulationType;
```

Comment:
buildUserOp only calls eth_estimateUserOpGas on the bundler. so we probably should not have this here..
if smartWallet.sendTransaction calls buildUserOp and then sendUserop / sendSignedUserOp this only needs to be added in sendTransaction params as dto.

Reply:
Yea, will remove it and add it as a separate optional param on sendTransaction or I can create a dto and use that as a param on sendTransaction which would look better but will change the way users will need to provide params. It's better to not add this in v4